cmd/controller-manager/app/options/insecure_serving.go

```diff
@@ -92,8 +85,12 @@ func (s *InsecureServingOptions) ApplyTo(c **genericcontrollermanager.InsecureSe
 
 	if s.Listener == nil {
 		var err error
+		listen := options.CreateListener
```

Comment:
could you move it to else part in the if below?

Reply:
Would leave this short here, avoiding a declaration.